### ISSUE TITLE
Issue with Coinbase API grabbing Quote Currency

### ISSUE THREAD

**Issue by zahlernick** [whittlem/pycryptobot#832]
Hi all, I recently was using Coinbase Pro API and tried to switch over to the regular Coinbase API (Coinbase Advanced) and it appears to take the price of whatever crypto I'm trying to buy/sell, divide that by the funds in my Coinbase wallet, then tell me whatever the result is as my actual. (for example, $200 in wallet, with ETH $1600 per token, it will tell me that 200/1600 = 0.125 is my actual, and that my minimum is $200 when it's set to something else).

Is this because they updated the API? Is there a fix to this?

**Comment by ahmedrabii** [comment]
Coinbase API (Coinbase Advanced) is just not working it needs to be fixed : 

- scanner.py line 37 still getting api keys from configuration
- scanner line 46 get_markets_24hr_stats function does not exist for coinbase AuthAPI 
- AppState.py line 223 is getting the min as the total (your case)
- coinbase pro valid urls must be updated and refactored (same array too many places)
- matplotlib must be updated to latest version and also the used styles
- logbuysellinjson even when you set it to 1 it does not log orders into a json file 

With  **AppState.py line 223 is getting the min as the total (your case)**  fixed by replacing the line with base_min = float(0) the app seems to work but guess what  **no order is being fulfilled**  it tells you that an order has been made but nothing actually happened 

Basically I listed all bugs that i found in the hope that they will be fixed soon 😄
